azure-pipelines.yml

```diff
@@ -3,22 +3,26 @@
 
 # Build STL targeting x86, x64, arm, arm64
 
+variables:
+  agentPool: 'StlBuild-2020-03-28'
+  vcpkgLocation: '$(Build.SourcesDirectory)/vcpkg'
+
 stages:
   - stage: Code_Format_Validation
     displayName: "Code Format Validation"
     jobs:
       - job: Run_Code_Format_Validation
         timeoutInMinutes: 90
-        displayName: "Run Code Format Validation"
+        displayName: "Run"

```

Comment:
We need to write something otherwise it will show up as "Run_Code_Format_Validation".

Reply:
Maybe change "Run Validation" to "Validation" and "Code Format Validation" to "Code Format" because "Code Format Validation Run Validation" in github checks looks quite silly.